refinery/core/api.py

```diff
@@ -518,15 +518,17 @@ def dehydrate(self, bundle):
             except:
                 pass
 
-        isa_archive = bundle.obj.get_isa_archive()
-        pre_isa_archive = bundle.obj.get_pre_isa_archive()
-
-        if isa_archive:
-            bundle.data["isa_archive"] = isa_archive.uuid
-            bundle.data["isa_archive_url"] = isa_archive.get_datafile_url()
-
-        if pre_isa_archive:
-            bundle.data["pre_isa_archive"] = pre_isa_archive.uuid
+        investigation = bundle.obj.get_investigation()
```

Comment:
This cuts ORM queries for FileStoreItems in half compared to the previous approach when GET-ing from api/v1/datasets/